index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Itumeleng Rantsane</title>
+        <link rel="stylesheet" type="text/css" href="styles/style.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;1,100;1,600&display=swap" rel="stylesheet">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+    </head>
+    <body>
+        <header>
+          <nav id="nav">
+            <a class="logo" href="">I.R</a>
+            
+            <nav><img src="images/Union.png" alt="Menu"></nav>
+               
+          </nav>
+        </header>
+        <section>
+           <h1>I'm Itumeleng. So glad to see you!</h1>
+            <p>
+                I'm a front-end developer and I specialize in responsive web design. 
+                I can help you build a product, 
+                feature or website. 
+                Look through some of my work and experience! 
+            </p>
+            <footer>
+                <h3>LET'S CONNECT</h3>
+                <div>
+                   <ul>
+                     <li class="icon"><a href="https://twitter.com/irantsane"><img src="images/twitter.png" alt="twitter link"></a></li>
+                     <li class="icon"><a href="https://www.linkedin.com/in/itumeleng-rantsane-552a44244/"><img src="images/Linkedin_icon.png" alt="linked link"></a></li>
+                     <li class="icon"><a href=""><img src="images/M.png" alt="M"></a> </li>
+                     <li class="icon"><a href="https://github.com/TumiRantsane"><img src="images/github.png" alt="github link"></a></li>
+                     <li class="icon"><a href=""><img src="images/doits.png" alt="doit"></a></li>
+                   </ul>
+                </div>
+            </footer>
```

Comment:
- [ ] Kindly remove the `footer` tag , your social icons list are part of "Headline" section So Please try to remove the `footer` tag.